modules/react/button/stories/button/Button.stories.mdx

```diff
@@ -71,6 +72,12 @@ on a dark or colorful background such as `blueberry400`.
 
 <ExampleCodeBlock code={Delete} />
 
+### Grow Prop
+
+The example below shows the use of the `grow` prop on different variants of buttons. This will set the width of the button to the width of the container.
```

Comment:
```suggestion
The example below shows the use of the `grow` prop on different variants of buttons. This will set the width of the button to the width of its container.
```